npm-shrinkwrap.json

```diff
@@ -803,12 +803,12 @@
     },
     "less": {
```

Comment:
I regenerated shrinkwrap as part of the test love, less was leftover in there, sorry for the confusion.